cmd/cli/devstack/devstack.go

```diff
@@ -66,6 +66,7 @@ func NewCmd() *cobra.Command {
 	ODs := newDevStackOptions()
 	IsNoop := false
 	devstackFlags := map[string][]configflags.Definition{
+		"local_publisher":       configflags.LocalPublisherFlags,
```

Comment:
these configs are at the node level and not stack level. Should we drop them from here and just let devstack choose the ports?